webcomponents/vectortile-view-component/package.json

```diff
@@ -4,12 +4,15 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
-    "build": "npm run format && npm run lint && ng build --output-hashing none",
+    "build": "npm run generate:api && npm run format && npm run lint  &&  ng build --output-hashing none",
     "watch": "ng build --watch --configuration development",
     "test": "ng test",
-    "deploy": "ng deploy --base-href=https://pdok.github.io/gokoala/ --name=JLSchaap --email=John.Schaap@Kadaster.nl",
+    "deploy": "npm run generate:api && npm run format && npm run lint  && ng deploy --base-href=https://pdok.github.io/gokoala/ --name=JLSchaap --email=John.Schaap@Kadaster.nl",
     "lint": "ng lint",
-    "format": "npx prettier --write ."
+    "format": "npx prettier --write .",
+    "generate:api2": "openapi-generator-cli generate -i https://api.pdok.nl/lv/bgt/ogc/v1_0-preprod/api?f=json -g typescript-angular -o src/app/openapi",
+    "generate:api": "openapi --input 'https://api.pdok.nl/lv/bgt/ogc/v1_0-preprod/api?f=json' --output src/app/openapi --client angular",
```

Comment:
waarom verwijzen naar een pdok productie url? Dat lijkt mij niet gewenst. Alle openapi specs zijn ook beschikbaar in deze codebase, dus we kunnen beter de lokale specs gebruiken. Nu zijn dat wel templates, maar daar kunnen we mogelijk wat op verzinnen. Zou iig niet naar iets buiten deze codebase verwijzen.

Daarnaast: is dit wel generiek? we willen de viewer voor alle features gebruiken, niet alleen de BGT. En we genereren nu code specifiek uit de BGT OAF (met BGT specifieke collections).

Reply:
Is inderdaad niet generiek, openapi code gen verwijderd en (generieke) types handmatig toegevoegd